test/setup/deployers/operator-openshift.ts

```diff
@@ -1,3 +1,5 @@
+import { readFileSync, writeFileSync } from 'fs';
```

Comment:
Have we standardised on single vs double quotes? I've seen both :)

Reply:
It all differs across repos 😄  For this we started with single quotes and we've kept it that way. Ideally this should be solved/enforced by a formatting tool but that's on our backlog